os/syscall/syscall_lookup.h

```diff
@@ -102,8 +102,14 @@ SYSCALL_LOOKUP(pgalloc,                   2, STUB_pgalloc)
 #endif
 SYSCALL_LOOKUP(task_delete,               1, STUB_task_delete)
 SYSCALL_LOOKUP(task_restart,              1, STUB_task_restart)
+SYSCALL_LOOKUP(task_setcancelstate,       2, STUB_task_setcancelstate)
+SYSCALL_LOOKUP(task_testcancel,           0, STUB_task_testcancel)
 SYSCALL_LOOKUP(up_assert,                 2, STUB_up_assert)
 
+#  ifdef CONFIG_CANCELLATION_POINTS
```

Comment:
I think syscall_lookup.h and syscall.h are must same config. but syscall.h have not CONFIG_CANCELLATION_POINTS.